packages/graph/clients/typescript/api.ts

```diff
@@ -2078,23 +2078,36 @@ export interface UnresolvedDecisionTimeImage {
   axis: DecisionTime;
   /**
    *
-   * @type {UnresolvedDecisionTimeImageEnd}
+   * @type {TimeIntervalBound}
    * @memberof UnresolvedDecisionTimeImage
    */
-  end: UnresolvedDecisionTimeImageEnd | null;
+  end: TimeIntervalBound | null;
   /**
    *
-   * @type {UnresolvedDecisionTimeImageEnd}
+   * @type {TimeIntervalBound}
    * @memberof UnresolvedDecisionTimeImage
    */
-  start: UnresolvedDecisionTimeImageEnd | null;
+  start: TimeIntervalBound | null;
 }
 /**
- * @type UnresolvedDecisionTimeImageEnd
+ *
  * @export
+ * @interface UnresolvedDecisionTimeImageAllOf
  */
-export type UnresolvedDecisionTimeImageEnd = TimeIntervalBound;
-
+export interface UnresolvedDecisionTimeImageAllOf {
```

Comment:
This seems unused?

Reply:
I don't know how you can comment on that line, possible via an old commit in this PR, but this line is not touched by the PR anymore. However, you are right. That's the same issue as https://github.com/hashintel/hash/pull/1738#discussion_r1063429672